src/cuga/backend/cuga_graph/nodes/cuga_lite/tests/test_tool_calling_sdk_integration.py

```
@@ -0,0 +1,98 @@
+NOTIFIED: List[str] = []
+
+
+@tool
+def notify(customer: str) -> str:
+    """Send a notification to a customer."""
+    NOTIFIED.append(customer)
+    return f"notification sent to {customer}"
+
+
+class _ScriptedFCModel(BaseChatModel):
+    script: List[AIMessage] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)